sql/core/src/test/resources/sql-tests/inputs/transform.sql

```diff
@@ -206,7 +206,7 @@ FROM script_trans
 LIMIT 1;
 
 SELECT TRANSFORM(
-  b AS d5, a,
```

Comment:
what was the behavior before (Spark 3.1 or older) when there is an alias? do we just ignore the alias?

Reply:
> what was the behavior before (Spark 3.1 or older) when there is an alias? do we just ignore the alias?

Treat it just like select clause since `TRANSFORM`'s input child plan can be all kind of  `SELECT` clause

Reply:
Can we add an item in the migration guide? Mentioning that alias is not allowed anymore in `TRANSFORM`.

Reply:
> Can we add an item in the migration guide? Mentioning that alias is not allowed anymore in `TRANSFORM`.

Done